src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterMapperFactory.java

```diff
@@ -10,11 +10,14 @@
 import java.time.ZonedDateTime;
 import java.util.UUID;
 
-@SuppressWarnings("HideUtilityClassConstructor")
 final class LetterMapperFactory {
 
     public static final RowMapper<LetterStatus> LETTER_STATUS_MAPPER = new LetterStatusMapper();
 
+    private LetterMapperFactory() {
+        // utility class constructor
+    }
+
```

Comment:
so sonarqube complains. bummer

Reply:
I'm sure we could just use an extra suppression code